app/models/ebean/ReportRecipientGroup.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2018 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.ebean;
+
+import io.ebean.annotation.CreatedTimestamp;
+import io.ebean.annotation.PrivateOwned;
+import io.ebean.annotation.UpdatedTimestamp;
+
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.UUID;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+/**
+ * Data Model for SQL storage of groups of report recipients.
+ * <p>
+ * A recipient group can contain multiple {@link models.ebean.ReportRecipient} instances.
+ *
+ * @author Christian Briones (cbriones at dropbox dot com)
+ */
+@Entity
+@Table(name = "report_recipient_groups", schema = "portal")
+// CHECKSTYLE.OFF: MemberNameCheck
```

Comment:
The class above has `// CHECKSTYLE.ON: FinalClassCheck`,  but this does not.

Reply:
Noted, although I'm not sure why Checkstyle isn't complaining about this instance.